spec.md

```diff
@@ -1873,6 +1906,19 @@ message Snapshot {
   // `volume_content_source` in a `CreateVolumeRequest`. The default
   // value is false. This field is REQUIRED.
   bool ready_to_use = 5;
+
+  // The ID of the volume group snapshot that this snapshot is part of.
+  // It uniquely identifies the group snapshot on the storage system.
+  // This field is OPTIONAL.
+  // If this snapshot is a member of the volume group snapshot, the SP
+  // SHOULD provide the ID of the volume group snapshot in this field.
+  // If provided, CO MUST use this field to indicate that this snapshot
+  // is part of the specified group snapshot.
```

Comment:
Based on `If not provided`, should this only be required if separate deletion is not supported? e.g.:

```suggestion
  // If this snapshot is a member of a volume group snapshot, and it
  // CAN NOT be deleted as a stand alone volume, then the SP
  // MUST provide the ID of the volume group snapshot in this field.
  // If provided, CO MUST use this field in subsequent snapshot volume 
  // group operations to indicate that this snapshot is part of the
  // specified group snapshot.
```

Reply:
If you agree with this, should reword the additions under `DeleteSnapshot` as well.

Reply:
Updated.

I have the following in the DeleteSnapshot section:
```
Snapshots that are members of a group snapshot will be deleted when the group snapshot is deleted. The CO SHOULD NOT call this RPC with a snapshot_id for a snapshot that was created as part of a group snapshot, i.e. if the snapshot had a non-empty group_snapshot_id field at creation time. The SP MAY refuse to delete such snapshots with this RPC call and return an error instead.
```
Is there anything else you want me to add?